vignettes/including-static-dashboards.Rmd

```diff
@@ -0,0 +1,103 @@
+
+You can host Flexdashboards on Crunch's Shiny infrastructure using Shiny's `includeHTML` function. What `includeHTML` does is add an HTML file to the shiny UI. If all you include in the shiny UI is the Flexdashboard html file, then the Shiny app effectively becomes a static site. 
+
+### Basic workflow
+Follow these steps to include a Flexdashboard in your Shiny app: 
+1) First create a new Rmarkdown document with a Flexdashboard output option. 
+1) Render the Rmd file into an html file
+1) Uploaded the rendered html file to your app's Jupyter folder
+1) In your shiny UI code add `includeHTML(<filename>)` to the `ui` object. Your `app.R` file might look something like this:
+
+```
+library(shiny)
+library(crunchy)
+# Render the dashboard
+ui <- fluidPage(
+    includeHTML("dashboard")
+)
+
+server <- function(input, output, session) {}
+
+# Run the application 
+shinyApp(ui = ui, server = server)
+```
+
+### Conditional dashboards
+
+Another great use of Flexdashboards is easily produce different dashboards for different users. To do this we'll make use of Shiny's `renderUI` function to conditionally create a different UI for different types of users.
+
+```
+ui <- fluidPage(
+    uiOutput("dashboard")
+)
+```
+
+This creates the ui object, but most of the work is created on the server side. We can then use the `shinyUser()` function to get a user record for the current viewer of the app. This record includes some useful pieces of information, including the user's name, their email address, and their Crunch preferences. You can use this object to conditionally display different dashboards for different users like so: 
+
+```
+server <- function(input, output, session) {
+    user <- getUser()
+    output$dashboard <- renderUI({
+        if (grepl("acme.com$", user()$email)) {
+            includeHTML("acme_dashboard.html")
+        } else {
+            includeHTML("globex_dashboard.html")
+        }
+    })
+    
+}
+```
+What this does is check the user's email address to identify clients from one company, and then displays a rendered dashboard for that company. If their email isn't a match, they get the Globex dashboard. You can see a full example app in the `/inst` directory. 
+
+
+### Managing Dashboard Renders
+
+The above code assumes that you are going to be re-rendering your Rmd files manually, and uploading the results to the Jupyter server. You can also re-render the Rmd files dynamically as part of the Shiny app by calling `rmarkdown::render()` somewhere in your server function. For example the following app re-renders the dashboard whenever clicks an action button. Note that for this to work you should use the [reactivePoll](https://shiny.rstudio.com/reference/shiny/1.0.2/reactivePoll.html) to ensure that the UI is refreshed after the file is regenerated. 
```

Comment:
Why do you need to poll? Why doesn't the button click trigger the event directly?